ios/Classes/SwiftFlutterPlugin.swift

```diff
@@ -70,4 +74,23 @@ public class SwiftFlutterPlugin: NSObject, Flutter.FlutterPlugin {
         }
         group.wait()
     }
+    
+    public func share(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        let client: Client = self.initClientFromFlutter(call, result: result)!
+        let args = call.arguments as! Dictionary<String, Any>
+        let recordType = args["type"] as! String
+        let readerID = args["reader_id"] as! String
+        
+        let group = DispatchGroup()
+        group.enter()
+        DispatchQueue.global().async {
+            client.share(type: recordType, readerId: UUID(uuidString: readerID)!) { (shareResult) in
+                shareResult.analysis(ifSuccess: { voidResultFromShare in
```

Comment:
After `client.share()` completes, `shareResult` is an `E3dbResult` object that contains a `cryptoError` object with an encoded string. 

You can confirm that records were shared with `readerId` via logging or dashboard UI.

Reply:
Flow through Flutter succeeds without error, and `e3db-swift` share test passes despite `cryptoError`, so will come back to `cryptoError` later